test/helpers/fixture_deploy_helper.rb

```diff
@@ -51,12 +51,14 @@ def deploy_dir(dir, wait: true, allow_protected_ns: false, prune: true, bindings
       current_sha: SecureRandom.hex(6),
       context: KubeclientHelper::MINIKUBE_CONTEXT,
       template_dir: dir,
-      wait_for_completion: wait,
-      allow_protected_ns: allow_protected_ns,
-      prune: prune,
+      logger: test_logger,
       bindings: bindings
     )
-    runner.run
+    runner.run(
+      skip_wait: !wait,
```

Comment:
I think even `opts.on("--skip-wait") { wait_for_completion = false }` would be better than `skip_wait` in the runner